packages/shipit-deploy/src/tasks/deploy/update.js

```diff
@@ -62,10 +63,7 @@ const updateTask = shipit => {
         rsync: '--del',
       }
       const rsyncFrom = shipit.config.rsyncFrom || shipit.workspace
-      const uploadDirPath = path.resolve(
-        rsyncFrom,
-        shipit.config.dirToCopy || '',
-      )
+      const uploadDirPath = p.default.resolve(rsyncFrom, shipit.config.dirToCopy || '');
```

Comment:
Sorry I misread the PR, the only problem for me is the `default` here. We should not need it.

Reply:
The current solution uses : 
var _posix = _interopRequireDefault(require("path2/posix"));
...
...
const uploadDirPath = _posix.default.resolve(rsyncFrom, shipit.config.dirToCopy || '');

Could please elaborate on your comment ?

Reply:
By current solution I am referring to the existing code in shipit repo (not the changes done by us)